src/Tools/BuildValidator/BuildConstructor.cs

```diff
@@ -59,53 +62,115 @@ public Compilation CreateCompilation(MetadataReader metadataReader, string name)
             throw new InvalidDataException("Did not find language in compilation options");
```

Comment:
This should be `Task.FromException` vs a direct `throw`. May be easier to make the method `async`

Reply:
Fixed